OpenSim/Actuators/MuscleFirstOrderActivationDynamicModel.cpp

```diff
@@ -81,78 +65,13 @@ void MuscleFirstOrderActivationDynamicModel::constructProperties()
     constructProperty_minimum_activation(0.01);
 }
 
-void MuscleFirstOrderActivationDynamicModel::buildModel()
-{
-    setObjectIsUpToDateWithProperties();
-}
-
-void MuscleFirstOrderActivationDynamicModel::ensureModelUpToDate()
-{
-    if(!isObjectUpToDateWithProperties()) {
-        buildModel();
-    }
-
-    // The name is not counted as a property but it can change, so it must be
-    // updated as well.
-    std::string name = getName();
-    setName(name);
```

Comment:
Seems like this was an attempt to update the name of this class in the case that the Muscle's name changed. However, I don't think that's what was actually happening.